src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -50,6 +53,15 @@
         new SecureRandom().nextBytes(secretKey);
     }
 
+    /*
+    This test fails during Java 17 build due to a known bug: https://bugs.openjdk.java.net/browse/JDK-8251547
+    TODO: This method should be removed once a fix is implemented
```

Comment:
Rather than create a TODO can you create an issue that captures what is wrong and what should be done?

Reply:
I have already created an issue and connected it to this PR as well

https://github.com/opensearch-project/security/issues/1620